### ISSUE TITLE
InstaPy Doesn't work - ISSUE WITH COOKIE

### ISSUE THREAD

**Issue by cormo1990** [InstaPy/InstaPy#2228]
@uluQulu any ideas?
below the traceback

Traceback (most recent call last):
  File "", line 11, in <module>
    session.login()
  File "/Users/Desktop/InstaPy_8/instapy/instapy.py", line 271, in login
    self.bypass_suspicious_attempt):
  File "/Users/Desktop/InstaPy_8/instapy/login_util.py", line 133, in login_user
    "//select[@class='_fsoey']/option[text()='English']").click()
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/webdriver.py", line 293, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/webdriver.py", line 752, in find_element
    'value': value})['value']
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//select[@class='_fsoey']/option[text()='English']"}
  (Session info: chrome=67.0.3396.79)
  (Driver info: chromedriver=2.37.544337 (8c0344a12e552148c185f7d5117db1f28d6c9e85),platform=Mac OS X 10.12.6 x86_64)

**Comment by cormo1990** [comment]
UPDATE @uluQulu 

I download the latest release of InstaPy and works, I solved the cookie problems but the page on browser seems corrupted and don't look well(see the pic attached).
I want to know if the function "non_watchers=True" is implemented in the latest release.
can you answer bro? thanks a lot

![schermata 2018-06-12 alle 19 25 20](https://user-images.githubusercontent.com/36590321/41306427-67e60510-6e76-11e8-8782-46b935ede83a.png)

**Comment by uluQulu** [comment]
Hi @cormo1990 
Actually I saw your issue but to suggest something I had to look deeper and I am really  busy with the latest implementation to finalize.

It's great that it works again!

BTW I didn't get what looks corrupted.

Well the last implementation I will share soon brings lots of capabilities will be unique to InstaPy and `non_watchers` feature will also be able to supported easily by all of the features, not just by `follow_user_followers` as we did in old thread.

But in this step I did not implement it but after the initial code passes, I will do it in the next phase.
It's so easy to do, but first I want people understand new tools and then add some other functionalities like `non_watchers`.

See PRs, maybe today I share latest code.
